src/librbd/api/Group.cc

```diff
@@ -1242,6 +1242,63 @@ int Group<I>::snap_list(librados::IoCtx& group_ioctx, const char *group_name,
   return 0;
 }
 
+template <typename I>
+int Group<I>::image_snap_list(librados::IoCtx& group_ioctx, const char *group_name,
+			const char *group_snap_name, std::string &group_snap_id,
+      std::vector<group_image_snap_info_t> *image_snaps)
+{
+  CephContext *cct = (CephContext *)group_ioctx.cct();
+  ldout(cct, 20) << "io_ctx=" << &group_ioctx
+		 << " group name " << group_name
+     << ", group snap name " << (group_snap_name ? group_snap_name : "") << dendl;
+
+  int r = 0;
+  if (group_snap_name)
+  {
```

Comment:
might want to move `{` to the end of previous line to be consistent with the coding style of the rest of this source file.